modules/ROOT/pages/servers-create.adoc

```diff
@@ -26,23 +30,25 @@ To add a Mule server to Runtime Manager:
 . From Anypoint Platform, select *Runtime Manager*.
 . Click *Servers* in the left menu.
 . Click the *Add Server* button.
+. Enter a name for your server.
++
+Server names can contain up to 60 alphanumeric characters (a-z, A-Z, 0-9), periods (.), hyphens (-), and underscores (_), but not spaces or other special characters.
+Runtime Manager supports Unicode characters in server names.
```

Comment:
is this true:
```suggestion
Server names can contain up to 60 Unicode or alphanumeric characters (a-z, A-Z, 0-9), periods (.), hyphens (-), and underscores (_), but not spaces or other special characters.
```

Reply:
How do we work around that ^ transforming into italic because of the underscore?

Reply:
This didn't change - it just moved - and it currently does not. See https://docs.mulesoft.com/runtime-manager/servers-create#add-a-server